apps/emqx_dashboard/src/emqx_dashboard_swagger.erl

```diff
@@ -84,13 +84,15 @@ fields(limit) ->
     Meta = #{in => query, desc => Desc, default => ?DEFAULT_ROW, example => 50},
     [{limit, hoconsc:mk(range(1, ?MAX_ROW_LIMIT), Meta)}].
 
--spec(schema_with_example(hocon_schema:type(), term()) -> hocon_schema:field_schema_map()).
-schema_with_example(Type, Example) ->
-    hoconsc:mk(Type, #{examples => #{<<"example">> => Example}}).
-
--spec(schema_with_examples(hocon_schema:type(), map()) -> hocon_schema:field_schema_map()).
-schema_with_examples(Type, Examples) ->
-    hoconsc:mk(Type, #{examples => #{<<"examples">> => Examples}}).
+-spec(schema_with_meta(hocon_schema:type(), map()) -> hocon_schema:field_schema_map()).
+schema_with_meta(Type, Meta0) ->
```

Comment:
I recommend keeping this method rather than just removing it. There are two advantages to doing so
1. it retains the convenience function for the caller to use directly.
2. No need to modify other modules that reference this function.

E.g:
```
schema_with_example(Type, Example) ->
    schema(Type, #{example => Example}).

schema_with_examples(Type, Examples) ->
    schema(Type, #{examples => Examples}).

-spec schema(Type :: hocon_schema:type(),
                         Meta :: #{desc => string(),
                                         example => term(),
                                         examples => map()}) ->  hocon_schema:field_schema_map().
```